### ISSUE TITLE
Could not load filesystem dll. Filesystem crashed during reconstruction.

### ISSUE THREAD

**Issue by ank81** [LAGonauta/MetaAudio#14]
Hi,

I am getting the above error when trying to start Blue Shift or Opposing Force. Using it with soft OpenAL. Folder structure seems to be correct. What could be the issue?

Thank you

**Comment by ank81** [comment]
Think it could be related the Oct. 25th update.

**Comment by LAGonauta** [comment]
Does Half-Life itself works? Any problem with other mods?

**Comment by ank81** [comment]
The issue seems to have resolved itself without my input, they may slipped another unannounced update yesterday evening or something. As for the issue, HL and MetaAudio were working flawlessly a month ago, straight out of the box. I decided to reinstall the game yesterday and noticed that they've introduced two updates since, on the 8th and 25th of october. One of these claims to have "Fixed mods not loading when they use certain GameDLL paths". Yesterday upon downloading HL I started to experience several issues with the vanilla game straight away, no MetaAudio involved. The game would not launch 90% of the times I tried starting it, simply nothing would happen, but there would be a ~1900kb hl.exe process hanging in task manager and steam would report the game as running, but wouldn't close no matter how many times I'd click on that button. Even today the game is still acting up, when I exit it steam still says that it's running, even though no process is present in the task manager, and even if I restart the steam it'll still say that the game is running. At least MetaAudio is working now. Thank you for your work on this, it's the greatest thing to come along since A3D, sounds way better than it too.
